lib/tasks/taxonomy/validate_taxons_base_paths.rake

```diff
@@ -14,9 +14,20 @@ namespace :taxonomy do
     if base_path_checker.invalid_taxons.any?
       puts "-" * 36
 
-      puts "The following taxons do not follow the taxon URL structure:"
+      puts "The following taxons did not match the taxon URL structure. Attempting to fix this..."
```

Comment:
I think it would be useful to make this optional, and opt in, e.g. `rake taxonomy:validate_taxons_base_paths[and_fix]` just so we can still run this without it doing anything.